CONTRIBUTING.md

```diff
@@ -99,3 +99,14 @@ make -f posix.mak apidocs-prerelease
 
 The output is in `$R/dlang.org/web/phobos-prerelease` and
 `$R/dlang.org/web/library-prerelease`.
+
+
+### Writing style guide
+
+* Use the _present_ tense (except for historic facts)
+* Use a _neutral_ tone (No strong narrator = no first-person ("I", "we") nor second-person ("you"))
+* Use a _formal_ style (No slang)
+* Use the [American English spelling](https://en.wikipedia.org/wiki/American_and_British_English_spelling_differences)
+* Use the [Oxford comma](https://en.wikipedia.org/wiki/Serial_comma) for coordinating conjunctions (C, C++__,__ and D)
```

Comment:
> Ah you are emphasizing the comma. :) All good.

Yep ;-)
For the future:  GitHub also allows to directly preview Markdown. Just click on the "View" link (top right).